src/Symfony/Bridge/PhpUnit/bin/simple-phpunit

```diff
@@ -15,6 +15,13 @@
 
 error_reporting(-1);
 
+function passthru_or_fail($command) {
```

Comment:
this must be a closure: we're careful to not create any global symbol in the script.

Reply:
$passthru = function ($command) {...}